server/src/test/java/org/elasticsearch/rest/RestHttpResponseHeadersTests.java

```diff
@@ -49,8 +50,8 @@ public void testUnsupportedMethodResponseHttpHeader() throws Exception {
          * method list, passing in the RandomizedContext's Random instance,
          * before picking out a candidate sublist.
          */
-        List<RestRequest.Method> validHttpMethodArray = new ArrayList<RestRequest.Method>(Arrays.asList(RestRequest.Method.values()));
```

Comment:
just a style question. Was replacing `RestRequest.Method` with `Method` and an import intentional? I dont' have strong opinion agains this, but using `RestRequest.Method` was more explicit that this lives within RestRequest. I am not sure the context of the `Method` class is that obvious from here.
It would be the same for common usage of `Map.Entry` vs `Entry`
This applies to other places in the PR where an import was for nested class. 
Also it looks like this is probably not documented, but a convention in ES (correct me if my observation is wrong)

Reply:
It was intentional, I think it's a bit prettier in this case, but it's not a huge deal to me and I'll happily back it out if you'd rather I left it alone.

Reply:
i think it would be better to refactor the nested classes imports separately. This would help to make the PR smaller (we are touching a lot of places with that refactoring, not only related to RestRequest.METHOD)

Reply:
7bb0919 covers this, I twiddled things back.